ee/app/license/server/license.ts

```diff
@@ -76,6 +79,12 @@ class LicenseClass {
 		});
 
 		this.validate();
+
+		if (this.hasAnyValidLicense()) {
```

Comment:
should we change this to be called only once? right now it can be called multiple times, for every time valid license added..